docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -39,6 +40,7 @@ The best device is chosen using the following logic:
 | 4        || Intel® CPU                                          | FP32, FP16, INT8, BIN               |
 |          || (e.g. Intel® Core™ i7-1165G7)                       |                                     |
 +----------+------------------------------------------------------+-------------------------------------+
+@endsphinxdirective
 
```

Comment:
Why do we need this pragma? I suppose markdown has tables support without sphinx help

Reply:
Yes it does, although the sphinx grid-tables give nice options of formatting with line breaks and fixed widths and stuff. This particular formatting (with bars and plusses) does not work for .md so I had to put it in sphinxdirectives.